### PR TITLE
[WIP] Update version numbers and timeouts to fix the CI

### PR DESCRIPTION
**Pull request by homernajafi**

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -55,10 +55,10 @@ export NEW_GKE_VER=
...
+#    create = "40m" // default 30m
+#    update = "25m" // default 10m
+#    delete = "25m" // default 10m
 #  }
-export TIMEOUT_CREATE="30m"
-export TIMEOUT_UPDATE="15m"
-export TIMEOUT_DELETE="15m"
+export TIMEOUT_CREATE="40m"
+export TIMEOUT_UPDATE="25m"
+export TIMEOUT_DELETE="25m"
```
[the remark this answers is not in the document]
Comment (homernajafi) [reply env]: Test this please

**Comment by homernajafi** [comment]
Test this please

**Comment by chrislovecnm** [comment]
test this please

**Comment by homernajafi** [comment]
Test this please

**Comment by homernajafi** [comment]
Test this please

**Comment by homernajafi** [comment]
Test this please

**Comment by homernajafi** [comment]
Test this please

**Comment by homernajafi** [comment]
test this please

**Comment by homernajafi** [comment]
test this please

**Comment by homernajafi** [comment]
test this please

**Comment by homernajafi** [comment]
test this please

**Pull request merged**